CTN-001.tex

```diff
@@ -24,7 +24,7 @@
 % \setDocSubtitle{A subtitle}
 
 \author{%
-Andrés A. Plazas Malagón, Seth Digel, Aaron Roodman, Alex Broughton, the LSST Camera Team. 
+Andrés A. Plazas Malagón, Seth Digel, Aaron Roodman, Alex Broughton, HyeYun Park, Yousuke Utsumi, and the LSST Camera Team. 
```

Comment:
I don't think you rebased before you started. The way authors are handled is completely different now because I fixed things up to the current style in #8
There is now an `authors.yaml` where you add the author IDs.